From: Jan Engelhardt <jengelh@inai.de>
Date: 2014-12-18 11:43:21.385084191 +0100
Refresh: Wed Dec  2 14:19:14 CET 2015

Author could not decide between "sdlbgifast" and "bgifast"
(same for bgislow), so add extra symbols to enable both names.

---
 Makefile.am      |    2 ++
 configure.ac     |    9 +++++++++
 src/Makefile.am  |    8 ++++++++
 test/Makefile.am |   11 +++++++++++
 4 files changed, 30 insertions(+)

Index: SDL_bgi-2.2.2/Makefile.am
===================================================================
--- /dev/null
+++ SDL_bgi-2.2.2/Makefile.am
@@ -0,0 +1,2 @@
+ACLOCAL_AMFLAGS = -I m4
+SUBDIRS = src test
Index: SDL_bgi-2.2.2/configure.ac
===================================================================
--- /dev/null
+++ SDL_bgi-2.2.2/configure.ac
@@ -0,0 +1,9 @@
+AC_INIT([SDL_bgi], [2.2.4])
+AC_CONFIG_AUX_DIR([build-aux])
+AC_CONFIG_MACRO_DIR([m4])
+AC_PROG_CC
+AM_INIT_AUTOMAKE([foreign subdir-objects tar-pax dist-xz no-dist-gzip])
+LT_INIT
+PKG_CHECK_MODULES([sdl2], [sdl2])
+AC_CONFIG_FILES([Makefile src/Makefile test/Makefile])
+AC_OUTPUT
Index: SDL_bgi-2.2.2/src/Makefile.am
===================================================================
--- /dev/null
+++ SDL_bgi-2.2.2/src/Makefile.am
@@ -0,0 +1,8 @@
+AM_CPPFLAGS = ${sdl2_CFLAGS}
+AM_CFLAGS = -Wall
+lib_LTLIBRARIES = libSDL_bgi.la
+libSDL_bgi_la_SOURCES = SDL_bgi.c
+libSDL_bgi_la_LIBADD = -lm ${sdl2_LIBS}
+libSDL_bgi_la_LDFLAGS = -no-undefined -version-info 2:0:0
+pkgincludedir = ${includedir}/SDL2
+pkginclude_HEADERS = SDL_bgi.h
Index: SDL_bgi-2.2.2/test/Makefile.am
===================================================================
--- /dev/null
+++ SDL_bgi-2.2.2/test/Makefile.am
@@ -0,0 +1,11 @@
+AM_CPPFLAGS = -I${top_srcdir}/src
+AM_CFLAGS = -Wall
+EXTRA_PROGRAMS = \
+	cellular dla fern floodfilltest hopalong kaleido life \
+	mandelbrot mousetest moveit plasma sdlbgidemo simple turtledemo
+LDADD = ../src/libSDL_bgi.la
+hopalong_LDADD = ${LDADD} -lm
+plasma_LDADD = ${LDADD} -lm
+sdlbgidemo_LDADD = ${LDADD} -lm
+turtledemo_SOURCES = turtle.c turtledemo.c
+turtledemo_LDADD = ${LDADD} -lm
